pkg/storage/mysql/mysql.go

```diff
@@ -161,9 +161,11 @@ func (m *MySQL) read(ctx context.Context, store string, tupleKey *openfgav1.Tupl
 		).
 		From("tuple").
 		Where(sq.Eq{"store": store})
-	if opts != nil {
+
+	if sortByUlid {
```

Comment:
no functional change here but it's more clear when we sort now (in ReadPage, not Read)